transloadit/resource_template_credential_test.go

```diff
@@ -0,0 +1,63 @@
+package transloadit
+
+import (
+	"fmt"
+	"math/rand"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccTemplateCredential_basic(t *testing.T) {
+
+	randInt := rand.Int() % 1000
```

Comment:
Rand.Int() will for every execution of the tests yield the same result because the seed is always the same at the start of the Go runtime. See the math/Rand package's description for more details.

If we want different numbers for each test run, we have to manually seed the number generator. Is that desired?

Reply:
As template credential have a unique name and we can have tests in parallel (multiple executions of CI on different branch), it is a choice to randomize it. And yes the seed is currently initialized from timestamp (https://github.com/transloadit/terraform-provider-transloadit/blob/master/transloadit/provider_test.go#L21) 
If you want to reproduce it, perhaps we could print the epoch that is used as seed

Reply:
> If you want to reproduce it, perhaps we could print the epoch that is used as seed

I actually do not want to reproduce it but to ensure that there is no naming collision for multiple test runs. The initialization using a timestamp-based seed is exactly what I was looking for :+1: